app/views/songs/index.html.erb

```diff
@@ -39,7 +39,9 @@
       serverSide: true,
       responsive: true,
       lengthChange: false,
-      dom: '',
+      pageLength: 25,
```

Comment:
Oh ok! Is it really that slow searching once per keystroke on your machine? It might be because you're running on Webrick instead of Unicorn locally.

Btw, I usually do big changes like this that are out of scope of the PR in a separate PR. Not just because to be strict about rules but because it helps you tackle one problem so you don't get overwhelmed. I'm actually gonna do that by opening two more PRs.